package.json

```diff
@@ -2821,6 +2821,131 @@
+												"name": "+Go to",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "g",
+														"name": "Peek definition",
+														"type": "command",
+														"command": "editor.action.peekDefinition"
+													},
+													{
+														"key": "d",
+														"name": "Peek declaration",
+														"type": "command",
+														"command": "editor.action.peekDeclaration"
+													},
+													{
+														"key": "r",
+														"name": "Peek references",
+														"type": "command",
+														"command": "editor.action.referencesSearch.trigger"
+													},
+													{
+														"key": "s",
+														"name": "Go to symbol in editor",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "a",
+														"name": "Switch Header/Source",
+														"type": "command",
+														"command": "C_Cpp.SwitchHeaderSource"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"type": "command",
+														"command": "workbench.actions.view.problems"
+													},
+													{
+														"key": "f",
+														"name": "Go to file in explorer",
+														"type": "command",
+														"command": "workbench.files.action.showActiveFileInExplorer"
+													},
+													{
+														"key": "G",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "D",
+														"name": "Go to declaration",
+														"type": "command",
+														"command": "editor.action.revealDeclaration"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "s",
+														"name": "Rename Symbol",
+														"type": "command",
+														"command": "editor.action.rename"
+													}
+												]
+											},
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format Document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "s",
+														"name": "Format Selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													}
```

Comment:
Is there any reason these are not following https://vspacecode.github.io/docs/conventions?
Note: not all LSP provide all format options. Make sure you test it and make sure c++ works with the options.

Reply:
I was oblivious to this existing thanks for calling it out. I've tested and updated them to match the conventions.